.gitignore

```diff
@@ -1,4 +1,5 @@
 Gemfile.lock
+gemfiles/*.lock
```

Comment:
I think these should stay so that the next person to pick it up gets the exact versions used in these changes

Reply:
Not sure I agree. 

It's a gem. This is not standard practice for gems... and our code SHOULD be resilient to any versions we're able to match.

Reply:
Also, we only added them last month... so this has never been a problem for us in the past afaik.